### PR TITLE
Updated AE Schema

### PR DESCRIPTION
**Pull request by jSylvestre**

### CODE REVIEW

File: Sloth.Core/Services/AggieEnterpriseService.cs [hunk 1]
```diff
@@ -156,8 +156,8 @@ public async Task<IGlJournalRequestResult> CreateJournal(Source source, Transact
         /// <param name="requestId">Found in JournalRequest.RequestId</param>
         /// <returns></returns>
         public async Task<IGlJournalRequestStatusResult> GetJournalStatus(string requestId)
```
Comment (srkirkland) [inline Sloth.Core/Services/AggieEnterpriseService.cs:158]: would a better solution be to just take a `Guid` as a param?

Comment (jSylvestre) [reply Sloth.Core/Services/AggieEnterpriseService.cs:158]: Less things to change... I think we are just storing it as a string. At least I was in OPP.

Comment (jSylvestre) [reply Sloth.Core/Services/AggieEnterpriseService.cs:158]: NM, it is a guid. I'll fix

**Review by srkirkland — APPROVED** [review]
yea i like that better

**Pull request merged**